inventory/any.cf

```diff
@@ -435,6 +443,118 @@ bundle agent cfe_autorun_inventory_cpuinfo
       "$(const.t) CPU physical cores: $(cpuinfo_physical_cores)";
 }
 
+bundle common cfe_autorun_inventory_aws
+# @brief inventory AWS EC2 instances
+#
+# Provides:
+#  ec2_instance class based on Amazon markers in dmidecode's system-uuid, bios-version or bios-vendor
+{
+  classes:
+    !disable_inventory_aws::
+      "ec2_instance" -> { "CFE-2924" }
+        comment => "See http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/identify_ec2_instances.html",
+        scope => "namespace",
+        expression => regcmp("^[eE][cC]2.*", "$(cfe_autorun_inventory_dmidecode.dmi[system-uuid])"),
```

Comment:
Would this be cleaner as a comparison of `EC2` to `string_upcase(string_head($(cfe_autorun_inventory_dmidecode.dmi[system-uuid]), 3)`? I'm just wondering...

Reply:
@atsaloli or @mikeweilgart may have a better view on it. If so, take care of it with a follow-up PR please.

Reply:
@tzz looks like LISP code ;)

Reply:
YES IT DOES :P